docassemble/AssemblyLine/data/questions/al_reminders.yml

```diff
@@ -94,4 +110,16 @@ code: |
       allow_cron = False # Stop calling cron on this session once all tasks performed
   else:
     allow_cron = False
-  response()
\ No newline at end of file
+  response()
+---
+code: |
+  if al_user_wants_reminders and task_not_yet_performed(f"al_reminder_initial_{reminder}"):
+    if al_user_preferred_reminder_formats.get("email") and task_not_yet_performed(f"al_reminder_initial_{reminder}_email"):
+      if not send_email(to=al_user_reminder_email, template=value(al_reminders[reminder]["initial email template"]), task=f"al_reminder_initial_{reminder}_email"):
+        log(f"{ all_variables(special='titles').get('title', AL_ORGANIZATION_TITLE) }: Couldn't send reminder to { showifdef('users[0]') } via {al_user_reminder_email}")
+    if al_user_preferred_reminder_formats.get("sms") and task_not_yet_performed(f"al_reminder_initial_{reminder}_sms"):
+      if not send_sms(to=al_user_reminder_phone, template=value(al_reminders[reminder]["initial sms template"]), task=f"al_reminder_initial_{reminder}_sms"):
+        log(f"{ all_variables(special='titles').get('title', AL_ORGANIZATION_TITLE) }: Couldn't send reminder to { showifdef('users[0]') } via {al_user_reminder_phone}")
+    mark_task_as_performed(f"al_reminder_initial_{reminder}")
+
+  al_reminders_sent_initial_test_message
```

Comment:
Is `al_reminders_sent_initial_test_message` supposed to be a variable used to trigger this block? If so, doesn't it need to be given a value here?